docs/docusaurus/versioned_docs/version-0.18/cloud/validations/manage_validations.md

```diff
@@ -72,6 +72,8 @@ To make the identification of Data Assets containing partitions easier, partitio
     
     - To view the run history of all Validations, select **All Runs** to view a graph showing the Validation run history for all columns.
 
+    - Note: items in run history with a calendar icon are run on a GX-managed schedule.
```

Comment:
As previously suggested:

```suggestion
:::tip

Items in run history with a calendar icon are run on a GX-managed schedule.

:::
```

If you want the admonition box to line up under the list for step 5, I believe you can do so by matching the indentation of the list items.  (With docusaurus, I believe that a 3 space indentation is used to place items "under" another element when it renders to html.)

Reply:
I tried to do this but I don't think it worked :/ Can't find a good example in the docusaurus docs